testsuites/syncgateway/functional/tests/test_log_rotation.py

```diff
@@ -0,0 +1,568 @@
+import json
+import os
+
+import pytest
+
+from keywords.ClusterKeywords import ClusterKeywords
+from keywords.RemoteExecutor import RemoteExecutor
+from keywords.SyncGateway import SyncGateway
+from keywords.SyncGateway import sync_gateway_config_path_for_mode
+from keywords.utils import log_info
+from libraries.testkit.cluster import Cluster
+
+
+@pytest.mark.sanity
+@pytest.mark.syncgateway
+@pytest.mark.parametrize("sg_conf_name", [
+    "custom_sync/sync_gateway_log_rotation"
```

Comment:
The config is not related to custom sync, can you pull it up a directory? Also, I think the prefix may be unnecessary (you could use `log_rotation_cc.json`). Recently, I've been trying to use simpler names to avoid redundancy.